### PR TITLE
nspr: Add CLANGARM64

### PR DESCRIPTION
**Pull request by hmartinez82**

### CODE REVIEW

File: mingw-w64-nspr/nspr-4.34_aarch64.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+diff -Naur nspr/pr/include/md/_win95.cfg nspr-c/pr/include/md/_win95.cfg
+--- nspr/pr/include/md/_win95.cfg	2022-07-28 21:07:56.185275300 -0600
++++ nspr-c/pr/include/md/_win95.cfg	2022-07-28 21:35:38.992454900 -0600
+@@ -68,7 +68,7 @@
+ #define PR_BYTES_PER_WORD_LOG2	2
+ #define PR_BYTES_PER_DWORD_LOG2	3
+ 
+-#elif defined(_M_X64) || defined(_M_AMD64) || defined(_AMD64_) || defined (_WIN64)
++#elif defined(_M_X64) || defined(_M_AMD64) || defined(_AMD64_) || defined (_WIN64) || defined (_M_ARM64)
```
Comment (Biswa96) [inline mingw-w64-nspr/nspr-4.34_aarch64.patch:9]: Are these changes required? 

> `_WIN64` Defined as 1 when the compilation target is 64-bit ARM or x64.

https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros

Comment (hmartinez82) [reply mingw-w64-nspr/nspr-4.34_aarch64.patch:9]: @Biswa96 you are right, fixing :)
